reporthandling/results/v1/resourcesresults/datastructures.go

```diff
@@ -11,9 +11,9 @@ import (
 type Result struct {
 	ResourceID string `json:"resourceID"` // <apigroup>/<namespace>/<kind>/<name>
 
-	AssociatedControls  []ResourceAssociatedControl        `json:"controls,omitempty"`
-	RawResource         reporthandling.Resource            `json:"rawResource,omitempty"`
-	PrioritizedResource prioritization.PrioritizedResource `json:"prioritizedResource,omitempty"`
+	AssociatedControls  []ResourceAssociatedControl         `json:"controls,omitempty"`
+	RawResource         *reporthandling.Resource            `json:"rawResource,omitempty"`
```

Comment:
Do we have an interface wrapping the two objects? If so, we should use the interface